x-pack/solutions/security/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -447,6 +447,17 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'mac.advanced.device_control.filter_images',
```

Comment:
Are these updated in the elasti.co docs as well?

Reply:
I will handle that. A bunch of advanced options added in 9.2 need online documentation updates.